ui/pages/reservation-unit/[...params].tsx

```diff
@@ -245,6 +249,7 @@ const ReservationUnitReservation = ({
     QueryReservationByPkArgs
   >(GET_RESERVATION, {
     variables: { pk: reservationData?.pk },
+    fetchPolicy: "no-cache",
```

Comment:
Don't like doing this on the client side. Might cause a ton of network requests if the component gets rerendered.
Is it absolutely necessary?